tokio/src/runtime/blocking/pool.rs

```diff
@@ -126,10 +129,15 @@ impl BlockingPool {
         shared.shutdown = true;
         shared.shutdown_tx = None;
         self.spawner.inner.condvar.notify_all();
+        let workers = shared.worker_threads.drain(0..).collect::<Vec<_>>();
 
         drop(shared);
 
-        self.shutdown_rx.wait(timeout);
+        if self.shutdown_rx.wait(timeout) {
+            for handle in workers {
+                handle.join().ok();
```

Comment:
Please don't use `.ok()` to ignore errors. Prefer to explicitly ignore them with an underscore:
```rust
let _ = handle.join();
```

Reply:
Fixed.